src/datanode/src/sql/copy_table.rs

```diff
@@ -52,7 +52,8 @@ impl SqlHandler {
             .context(error::TableScanExecSnafu)?;
         let stream = Box::pin(DfRecordBatchStreamAdapter::new(stream));
 
-        let accessor = Builder::default().build().unwrap();
+        // TODO: Remove hardcode and unwrap
+        let accessor = Builder::default().root("/").build().unwrap();
```

Comment:
Would you mind fixing the unwrap here?

Reply:
> Would you mind fixing the unwrap here?

I modified it to the following, but it looks ugly... should I define a new error?
```rust
let accessor = Builder::default()
    .root("/")
    .build()
    .context(error::InitBackendSnafu {
        config: Box::new(ObjectStoreConfig::File(FileConfig {
            data_dir: "/".to_string(),
        })),
    })?;
```